vtex-apps/order-items
# Fix item index to add to cart

#### What problem is this solving?
When trying to add a list of items to the cart when any of the items in the list are already in the cart, no items are added.



#### How to test it?


[Before](https://thalyta3--storecomponents.myvtex.com/blouse-with-knot/p)

[After](https://thalyta4--storecomponents.myvtex.com/blouse-with-knot/p)

- in the 'buy together' shelf, click the button to add the items to the cart.
- change the color of one or more items.
- try adding items to the cart again.
- the items that are not in the cart yet, should be added.

#### Screenshots or example usage:

Before:
![before](https://user-images.githubusercontent.com/20840671/89303820-11529880-d643-11ea-96a3-fe3c10a9a1bf.gif)

After:
![after](https://user-images.githubusercontent.com/20840671/89303827-13b4f280-d643-11ea-934c-da49b6329d78.gif)



#### Describe alternatives you've considered, if any.



#### Related to / Depends on

Depends on: https://github.com/vtex-apps/add-to-cart-button/pull/35


#### How does this PR make you feel? [:link:](http://giphy.com/)



![](put .gif link here - can be found under "advanced" on giphy)
